app/jobs/lakeshore/test_fail_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+class Lakeshore::TestFailJob < ActiveJob::Base
```

Comment:
Is this ever used anywhere else or is it just to test the logger? If so, you can move the job definition to the spec test itself:

``` ruby
describe "A failing job" do
  class TestFailJob < ActiveJob::Base
    [etc]
  end
  
  it "raises an error" do
    expect { TestJob.perform_now }.to raise_error(StandardError)
  end
end
```